enterprise/api/v1/serializers.py

```diff
@@ -914,3 +916,46 @@ def validate(self, data):  # pylint: disable=arguments-differ
             )
 
         return data
+
+
+class Base64EmailCSVField(serializers.Field):
```

Comment:
I'd put it in a separate fields file so people can find it for re-use, but that's a nit. How are we uploading other CSVs?

Reply:
Yeah can move this. Other CSVs are uploaded as part of a multipart/form outside of DRF.

Reply:
Oh, yeah, this approach is better. 🎉